### DISCUSSION TITLE
OUT OF GAS ALL THE TIME WHEN DEPLOYING ON RINKEBY

### DISCUSSION THREAD

**Discussion by FloUz-ai** [ProjectOpenSea/seaport#632] [Q&A]
HEY, 

Im trying to deploy on rinkeby the seaport js. my transaction failed all the time with a out of gas on etherscan. I dont get it... Any idea about what could be the reason. I've tried with an higher gas again and again but nothing works ... anyone had this issue?

**Comment by ryanio** [comment]
Seaport is already [deployed](https://github.com/ProjectOpenSea/seaport#deployments) on rinkeby at [0x00000000006c3852cbEf3e08E8dF289169EdE581](https://etherscan.io/address/0x00000000006c3852cbEf3e08E8dF289169EdE581#code)
